src/rapier_physics.rs

```diff
@@ -46,69 +46,65 @@ pub fn body_description_to_builder(
+            0 => RigidBody::Dynamic,
+            1 => RigidBody::Fixed,
+            2 => RigidBody::KinematicPositionBased,
+            3 => RigidBody::KinematicVelocityBased,
+            _ => RigidBody::Dynamic,
         };
 
+        //updated lock flags to store LockedAxes to how they are used in the newest version of rapier
+
         let lock_flags = {
-            let mut flags = RigidBodyMassPropsFlags::empty();
+            let mut flags = LockedAxes::empty();
             if body_desc.lock_translation.x != 0 {
-                flags.insert(RigidBodyMassPropsFlags::TRANSLATION_LOCKED_X);
+                flags.insert(LockedAxes::TRANSLATION_LOCKED_X);
             }
             if body_desc.lock_translation.y != 0 {
-                flags.insert(RigidBodyMassPropsFlags::TRANSLATION_LOCKED_Y);
+                flags.insert(LockedAxes::TRANSLATION_LOCKED_Y);
             }
             if body_desc.lock_translation.z != 0 {
-                flags.insert(RigidBodyMassPropsFlags::TRANSLATION_LOCKED_Z);
+                flags.insert(LockedAxes::TRANSLATION_LOCKED_Z);
             }
             if body_desc.lock_rotation.x != 0 {
-                flags.insert(RigidBodyMassPropsFlags::ROTATION_LOCKED_X);
+                flags.insert(LockedAxes::ROTATION_LOCKED_X);
             }
             if body_desc.lock_rotation.y != 0 {
-                flags.insert(RigidBodyMassPropsFlags::ROTATION_LOCKED_Y);
+                flags.insert(LockedAxes::ROTATION_LOCKED_Y);
             }
             if body_desc.lock_rotation.z != 0 {
-                flags.insert(RigidBodyMassPropsFlags::ROTATION_LOCKED_Z);
+                flags.insert(LockedAxes::ROTATION_LOCKED_Z);
             }
 
             flags
         };
 
-        let bundle = RigidBodyBundle {
-            body_type: RigidBodyTypeComponent(body_status),
-            position: RigidBodyPositionComponent(RigidBodyPosition {
-                position: isometry,
-                next_position: isometry,
-            }),
-            mass_properties: RigidBodyMassPropsComponent(RigidBodyMassProps {
-                flags: lock_flags,
-                ..Default::default()
-            }),
-            forces: RigidBodyForcesComponent(RigidBodyForces {
-                ..Default::default()
-            }),
-            activation: Default::default(),
-            damping: RigidBodyDampingComponent(RigidBodyDamping {
-                linear_damping: body_desc.damping_linear,
-                angular_damping: body_desc.damping_angular,
-            }),
-            ccd: RigidBodyCcdComponent(RigidBodyCcd {
-                ccd_enabled: body_desc.ccd_enable,
-                ..Default::default()
-            }),
-            ..Default::default()
-        };
-
-        commands
-            .entity(entity)
-            .insert_bundle(bundle)
-            .insert(RigidBodyPositionSync::Discrete);
+        //new method of inserting rigid bodies into entities
+        if body_desc.ccd_enable{
```

Comment:
If possible, could we get rid of this conditional here? I'd rather something like 
```
let initial_body = commands.entity(entity)
    .insert(body_status)

....
if body_desc.ccv_enable {
    initial_body.insert(Ccd::enabled())
}

```